src/lib/list/list-item.html

```diff
@@ -5,7 +5,13 @@
   </div>
 
   <ng-content
-      select="[md-list-avatar],[md-list-icon], [mat-list-avatar], [mat-list-icon]"></ng-content>
-  <div class="mat-list-text"><ng-content select="[md-line], [mat-line]"></ng-content></div>
+      select="[md-list-avatar], [md-list-icon], [mat-list-avatar], [mat-list-icon],
+              [mdListAvatar], [mdListIcon], [matListAvatar], [matListIcon]">
+  </ng-content>
+
+  <div class="mat-list-text">
```

Comment:
lines 12-14 have to be on one line so the ".mat-list-text:empty" css selector works.